plugins/outputs/elasticsearch/README.md

```diff
@@ -199,6 +199,13 @@ This plugin will format the events in the following way:
   ## If set to true a unique ID hash will be sent as sha256(concat(timestamp,measurement,series-hash)) string
   ## it will enable data resend and update metric points avoiding duplicated metrics with diferent id's
   force_document_id = false
+
+  ## Specifies the handling of NaN and Inf values.
+  ## This option can have the following values:
+  ##    none -- do not modify field-values (default); will produce an error if NaNs or infs are encountered
+  ##    drop -- drop fields containing NaNs or infs
+  ##    <any float> -- Replace NaNs and inf with the given number and -inf with the negative number
+  # float_handling = "none"
```

Comment:
Let's also have a separate setting for the replacement value. I think a setting that can be a string or a float will be a little confusing to users. They'll have to know it's a float, but really a string so it needs quotes.

We would also like eventually to replace or augment the toml config system with a remote config system. Settings that have multiple valid types makes that more difficult.

```suggestion
  ## Specifies the handling of NaN and Inf values.
  ## This option can have the following values:
  ##    none -- do not modify field-values (default); will produce an error if NaNs or infs are encountered
  ##    drop -- drop fields containing NaNs or infs
  ##    replace -- Replace NaNs and inf with the value in the setting 'float_replacement_value'.
  ##            Replace -inf with the value negated
  # float_handling = "none"
  # float_replacement_value = 9e99
```

Reply:
Done.